bitnami/kibana/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "kibana.fullname" . }}-test-connection"
+  labels: {{- include "kibana.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: wget
+      image: busybox
```

Comment:
this should be 'minideb' instead of `busybox`

Reply:
not done yet

Reply:
Done